### DISCUSSION TITLE
Change active background color for navbar

### DISCUSSION THREAD

**Discussion by ChristopherCalderon** [saadeghi/daisyui#2550] [Q&A]
![image](https://github.com/saadeghi/daisyui/assets/62729165/5244e082-e01a-477a-a229-4f1bee6c1cd8)

By default the navbar li elements get this background color when you click them, Ive tried a lot to change it but I cant find the property to do it
I can easily change the hover bg color with just ¨hover:bg-Red¨ for example, but I cant change it for the active state

**Comment by saadeghi** [comment]
Can you show your HTML?

**Reply by jakob-varenbud** [reply]
i have the exact same issue here. what ways do i have to override this? 

![Bildschirmfoto 2024-07-22 um 19 56 54](https://github.com/user-attachments/assets/050b6247-1e3f-462e-9875-7c544d3d3aac)

it is the "Navbar with menu and subemenu" component

**Comment by jakob-varenbud** [comment]
class="menu menu-horizontal px-1 [&_li>details>summary:active]:bg-red-500"
 
 here is the code with the  tailwind JIT syntax, to use the default theme and dont write a new one. 
 the color can be overwritten in the "neutral" color in the theme. this is where the bg-color comes from here, when the element set to :active
